etc/sync-function-validation-module.js

```diff
@@ -0,0 +1,550 @@
+function() {
```

Comment:
The functions and variables in this module are unchanged from when they were embedded directly in the sync function template file.